JSDemos/Demos/TreeList/UsingHeaderFilter/description.md

```diff
@@ -1 +1,7 @@
-This demo shows how to&nbsp;use the header filter for filtering data. The header filter contains a&nbsp;list of&nbsp;unique column values that you can select. To&nbsp;open this list, click the filter icon in&nbsp;a&nbsp;column header. Once you select a&nbsp;value from this list and click&nbsp;OK, the filter is&nbsp;applied, and the TreeList displays the full hierarchy down to&nbsp;matching rows, but not further.
\ No newline at end of file
+A header filter allows users to select column values from a predefined set and use the selected values to filter rows. To enable this behavior, set **headerFilter**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#visible) to **true**. This feature can be combined with the [filter row](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/UsingFilterRow).
+
+When you enable the header filter, a funnel icon appears in each column's header. Users can click this icon to open the drop-down list and choose values to filter by. You can use the **columns[]**.[headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/) property to configure individual header filters. If a certain header filter should be removed, set **columns[]**.[allowHeaderFiltering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHeaderFiltering) to **false** .
```

Comment:
Since this information is now in the paragraph above, I would to dedicate this paragraph to draw the distinction between common and individual header filter settings and highlighing the most valuable features.

```suggestion
To configure common header filter settings, use the [headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/) object. You can also specify the [headerFilter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/) object for individual columns. Common settings [enable search](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#allowSearch) within the popup menu and specify the menu's [width](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#width) and [height](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/#height). Individual settings also allow you to change auto-generated filter values. For example, you can specify a different [dataSource](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/#dataSource) for an individual header filter or group the filter values by a custom [groupInterval](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/headerFilter/#groupInterval).
```